itests/eth_fee_history_test.go

```diff
@@ -29,95 +46,109 @@ func TestEthFeeHistory(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	// Wait for the network to create 20 blocks
+	heads, err := client.ChainNotify(ctx)
+	require.NoError(err)
+
+	// Save the full view of the tipsets to calculate the answer when there are null rounds
+	tsHeights := []int{1}
+	go func() {
+		for chg := range heads {
+			for _, c := range chg {
+				tsHeights = append(tsHeights, int(c.Val.Height()))
+			}
+		}
+	}()
+
+	miner[0].InjectNulls(abi.ChainEpoch(1))
+	<-time.After(5 * blockTime)
+	miner[0].InjectNulls(abi.ChainEpoch(3))
+	<-time.After(2 * blockTime)
```

Comment:
Depending on time is usually flaky. Consider waiting for N number of tipsets produced again. See https://github.com/filecoin-project/lotus/pull/10427/files#diff-57c54de2d685e694dee103dc15a360776a5b30b5ec7fc77b5150c3756525d683R322-R328 for some hints.